plugin/src/main/java/io/snyk/eclipse/plugin/preferences/PreferencesPage.java

```diff
@@ -124,11 +150,30 @@ protected void createFieldEditors() {
 						+ "Please separate entries with \"" + File.pathSeparator + "\".",
 				getFieldEditorParent()));
 		StringFieldEditor trustedFoldersEditor = new StringFieldEditor(Preferences.TRUSTED_FOLDERS, "Trusted Folders:",
-				80, getFieldEditorParent());
+				WIDTH, getFieldEditorParent());
 		addField(trustedFoldersEditor);
 		disableSnykCodeIfOrgDisabled();
 	}
 
+	private SelectionAdapter authenticateSelectionAdapter() {
+		return new SelectionAdapter() {
+			@Override
+			public void widgetSelected(SelectionEvent e) {
+				performOk(); // This will perform an apply to the values in the PreferencePage and store
+								// these to the PreferenceStore.
+
+				String selectedAuth = getPreferenceStore().getString(Preferences.AUTHENTICATION_METHOD);
+				if (AuthConstants.AUTH_PERSONAL_ACCESS_TOKEN.equals(selectedAuth)) {
+					boolean launched = Program.launch(PAT_AUTHENTICATION_URL);
+					if (!launched) {
+						MessageDialog.openError(getShell(), "Open URL failed",
+								"Failed to launch PAT authentication URL.");
+					}
```

Comment:
this should work like IntelliJ & Visual Studio - it should just trigger the login command, which would do the rest. No logic in the IDE for this is needed.

Reply:
OAuth2 is not handled, yet, right? Just calling the `login` command should be enough and cover it, too.